run-test262.c

```diff
@@ -435,25 +441,54 @@ static JSValue js_evalScript_262(JSContext *ctx, JSValue this_val,
     return ret;
 }
 
-static void start_thread(pthread_t *thrd, void *(*start)(void *), void *arg)
+static void start_thread(js_thread_t *thrd, void *(*start)(void *), void *arg)
 {
+    // musl libc gives threads 80 kb stacks, much smaller than
+    // JS_DEFAULT_STACK_SIZE (256 kb)
+    static const unsigned stacksize = 2 << 20; // 2 MB, glibc default
+#ifdef _WIN32
+    *thrd = (HANDLE)_beginthread(stacksize, (void (*)(void *))(void *)start,
```

Comment:
How about adding js_thread to cutils, so we can eventually reuse it to implement workers?

Reply:
Fair enough!